core/src/main/java/org/bitcoinj/core/Bech32.java

```diff
@@ -103,10 +101,18 @@ public static String encode(final Bech32Data bech32) {
         return encode(bech32.hrp, bech32.data);
     }
 
+    private static void checkHRPLen(String hrp){
```

Comment:
I'm not convinced this is preferable to helpers like `checkArgument()`. Is there no JDK equivalent available?